packages/lesswrong/components/posts/PostsItem2.jsx

```diff
@@ -32,10 +32,15 @@ const styles = (theme) => ({
       opacity: .2,
     }
   },
+  fixedHeight: {
+    [theme.breakpoints.up('md')]: {
+      height: 48,
+    }
+  },
   postsItem: {
     display: "flex",
-    paddingTop: theme.spacing.unit*1.5,
-    paddingBottom: theme.spacing.unit*1.5,
+    paddingTop: 10,
```

Comment:
why 10?

Reply:
This affects the vertical position of elements other than the title (ie the karma, author, and comment-bubble), and they were a few pixels too low for some reason. This change moves them up. It does not affect the height of the PostsItem2 as a whole, which is fixed elsewhere.